ansible/roles/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
This was supposed to be deleted, botched a copy of an ansible directory, this is nginx tasks main, will remove